src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml

```diff
@@ -530,9 +533,21 @@
           <Tab text="Voter Error Rules">
             <VBox>
               <VBox styleClass="bordered-box" maxWidth="418.0">
-                <HBox alignment="CENTER_LEFT" spacing="4.0">
+                <HBox spacing="4.0">
                   <Label text="Overvote Rule *" prefWidth="160.0"/>
-                  <ChoiceBox fx:id="choiceOvervoteRule" prefWidth="220.0"/>
+                  <VBox spacing="4.0">
+                    <RadioButton mnemonicParsing="false" text="Always skip to next rank"
+                      toggleGroup="$overvoteRuleToggleGroup" fx:id="radioOvervoteAlwaysSkip"/>
+                    <RadioButton mnemonicParsing="false" text="Exhaust immediately"
+                      toggleGroup="$overvoteRuleToggleGroup"
+                      fx:id="radioOvervoteExhaustImmediately"/>
+                    <RadioButton mnemonicParsing="false" text="Exhaust if multiple continuing"
```

Comment:
Can these text properties just reference the strings already defined in the enum?

Reply:
Done, and also use these values for the validation error messages too.